### PR TITLE
[# 158678751] As a Looker embedded_role user, so that I can access dashboards after legacy access_filter_fields are disabled, **update the "filters" in Looker-hmac app to use access_filters and user_attributes.**

### PR DESCRIPTION
**Pull request by ajuvekar**
As a Looker embedded_role user, so that I can access dashboards after legacy access_filter_fields are disabled, **update the "filters" in Looker-hmac app to use access_filters and user_attributes.**

[Tracker #158678751](https://www.pivotaltracker.com/story/show/158678751)

### CODE REVIEW

**Comment by ajuvekar** [comment]
@nfoen , @richddr - Please CR.

**Comment by alexebird** [comment]
Still getting these emails from you all <3

**Comment by gupsydaisys** [comment]
Lol me too.

**Comment by soonahn** [comment]
You guys are giving away our secrets...

**Comment by wenever** [comment]
Lol

**Review by nfoen — COMMENTED** [review]
Hello everyone! 

@ajuvekar looks good to me. This should go with a PR in black, and we should also bump the version here.

**Comment by ajuvekar** [comment]
thanks @nfoen.

**Comment by soonahn** [comment]
@nfoen dinner next time you're in town?

File: lib/grnds/looker/hmac.rb [hunk 1]
```diff
@@ -25,6 +25,7 @@ def getUrl()
           json_permissions = @config[:permissions].to_json
           json_models = @config[:models].to_json
           json_access_filters = @config[:filters].to_json
+          json_user_attributes = @config[:user_attributes].to_json
```
Comment (cobbr2) [inline lib/grnds/looker/hmac.rb:28]: How are these set?

**Review by cobbr2 — CHANGES_REQUESTED** [review]
Once the code is ready (and it is), this PR needs to be completed by:
* Update the VERSION in lib/.../version.rb
* Pushing the rubygem version of it (there's a Rake task for publishing).
Then we'll need to update the version requirement in `black` to finish that PR up.

**Comment by cobbr2** [comment]
And for those of you out there wondering why you get updates... this is a public open-source repo, quite intentionally (we were the source of Looker's ruby example code, IIRC), and Github thinks you're still subscribed to it. Just drop your subscriptions if they're annoying :) Good to hear from you all!

File: lib/grnds/looker/hmac.rb (hunk 1 above)
Comment (cobbr2) [reply lib/grnds/looker/hmac.rb:28]: Oh. The caller is responsible for setting them, thus the `black` PR. Thanks, @ajuvekar .

**Comment by ajuvekar** [comment]
@cobbr2 - Updated the version in lib/.../hmac/version.rb to 0.1.0

**Review by nfoen — APPROVED** [review]

**Review by cobbr2 — APPROVED** [review]

**Pull request merged**